src/settings.rs

```diff
@@ -163,34 +339,299 @@ impl WasabiSettings {
+            )
+            .arg(
+                Arg::new("dont-limit-layers")
+                    .help("Do not apply the layer limit to the synth")
+                    .long_help(
+                        "This allows the synth to create as many layers as it \
+                        needs to play the MIDI file faithfully. Only turn this on if your \
+                        MIDI sounds bad, or your computer is running on GFuel",
+                    )
+                    .long("dont-limit-layers")
+                    .action(ArgAction::SetFalse),
+            )
+            .arg(
+                Arg::new("layer-count")
+                    .help("The amount of layers that the synthesizer can create")
+                    .long_help(
+                        "The amount of virtual pianos that the synth can create to \
+                        play your piece. A reasonable number for this value is the number of \
+                        CPU cores your computer has.",
+                    )
+                    .short('l')
+                    .long("layer-count")
+                    .value_parser(value_parser!(usize)),
+            )
+            .arg(
+                Arg::new("vel-ignore")
+                    .help("The range of note velocities that the synth will discard")
+                    .long_help(
+                        "Two numbers, comma seperated, that represent a range of velocities \
+                        that the synth will discard, making notes in the range inaudible.",
+                    )
+                    .short('v')
+                    .long("vel-ignore")
+                    .value_parser(range_parser),
+            )
+            .arg(
+                Arg::new("fade-out-kill")
+                    .help("Once a voice is killed, fade it out")
+                    .long_help(
+                        "Once the synthesizer kills one of it's voices, it will fade it \
+                        out as opposed to simply cutting it off",
+                    )
+                    .short('F')
+                    .long("fade-out-kill")
+                    .action(ArgAction::SetTrue),
+            )
+            .arg(
+                Arg::new("linear-envelope")
+                    .help("Adjust the rate of decay on the synth's voices")
+                    .long_help(
+                        "Switch the synth's voice's rate of decay from \
+                        exponential to linear. This may be bring a performance \
+                        improvement on some systems.",
+                    )
+                    .short('L')
+                    .long("linear-envelope")
+                    .action(ArgAction::SetTrue),
+            )
+            .arg(
+                Arg::new("no-effects")
+                    .help("Disable the synth's effects")
+                    .long_help(
+                        "Disable the effects that the synthesizer applies to the final audio \
```

Comment:
The limiter is not affected by this.
These effects only include those applied to individual voices (currently only high/low pass filter)